packages/lesswrong/components/ea-forum/TargetedJobAd.tsx

```diff
@@ -223,29 +224,137 @@ type EAGOccupation =
-  feedbackLinkPrefill: string,       // url param used to prefill part of the feedback form
-  bitlyLink: string,                 // bitly link to the job ad page
+  eagOccupations?: EAGOccupation[],                           // used to match on EAG experience + interests
+  interestedIn?: EAGOccupation[],                             // used to match on EAG interests
+  subscribedTagIds?: string[],                                // used to match on a set of topics that the user is subscribed to
+  readCoreTagIds?: string[],                                  // used to match on a set of core topics that the user has read frequently
+  coreTagReadsThreshold?: number,                             // used to adjust the threshold for how many post reads per topic to qualify for seeing the ad
+  logo: string,                                               // url for org logo
+  occupation: string,                                         // text displayed in the tooltip
+  feedbackLinkPrefill: string,                                // url param used to prefill part of the feedback form
+  bitlyLink: string,                                          // bitly link to the job ad page
   role: string,
-  insertThe?: boolean,               // set if you want to insert a "the" before the org name
+  insertThe?: boolean,                                        // set if you want to insert a "the" before the org name
   org: string,
-  orgLink: string,                   // internal link on the org name
+  orgLink: string,                                            // internal link on the org name
   salary?: string,
   location: string,
-  countryCode?: string,              // if provided, only show to users who we think are in this country
-  roleType?: string,                 // i.e. part-time, contract
-  deadline?: moment.Moment,          // also used to hide the ad after this date
-  getDescription: (classes: ClassesType) => JSX.Element
+  countryCode?: string,                                       // if provided, only show to users who we think are in this country
+  roleType?: string,                                          // i.e. part-time, contract
+  deadline?: moment.Moment,                                   // also used to hide the ad after this date
+  getDescription: (classes: ClassesType) => JSX.Element,
+  get80kDescription?: (classes: ClassesType) => JSX.Element   // just used on some ads for an A/B test
 }
 
 // job-specific data for the ad
 // (also used in the reminder email, so links in the description need to be absolute)
 export const JOB_AD_DATA: Record<string, JobAdData> = {
+  'gwwc-researcher': {
+    subscribedTagIds: [
+      'psBzwdY8ipfCeExJ7', // cause prioritization
+      'L6NqHZkLc4xZ7YtDr', // effective giving
+    ],
+    readCoreTagIds: [
+      'psBzwdY8ipfCeExJ7', // cause prioritization
+      'L6NqHZkLc4xZ7YtDr', // effective giving
+    ],
+    coreTagReadsThreshold: 60,
```

Comment:
I'm setting a higher threshold for this job because I don't want to show it to too many users for now. (A lot of users have read a lot of cause prio posts and effective giving posts.) We may decrease this later.